source/codegen/templates/service_helpers.mako

```diff
@@ -166,6 +166,10 @@ ${initialize_standard_input_param(function_name, parameter)}\
       ${c_type} ${parameter_name} = (${c_type})${request_snippet}.c_str();\
 % elif c_type == 'ViInt8[]' or c_type == 'ViChar[]':
       ${c_type_pointer} ${parameter_name} = (${c_type[:-2]}*)${request_snippet}.c_str();\
+% elif c_type == 'ViSession[]':
+      auto ${parameter_name}_request = ${request_snippet};
+      std::vector<${c_type_underlying_type}> ${parameter_name};
+      std::transform(${parameter_name}_request.begin(), ${parameter_name}_request.end(), std::back_inserter(${parameter_name}), [&](auto session) { return session_repository_->access_session(session.id(), session.name()); }); \
```

Comment:
This line is rather long. Consider putting params on their own lines.

Reply:
Split the line for this as well as ViBoolean[] case below.